latte.go

```diff
@@ -1,13 +1,15 @@
 package catppuccingo
 
-// latte variant.
+// Latte variant
 type latte struct{}
 
-// Latte flavour variant.
-var Latte Flavour = latte{}
+// Latte flavor variant
+var Latte Flavor = latte{}
```

Comment:
Just to confirm, this isn't going to cause an API breaking change right?

It's painfully obvious that I am not a Go programmer :smile:

Reply:
Disregard, read your updated PR description